AIPscan/Aggregator/mets_parse_helpers.py

```diff
@@ -44,14 +44,20 @@ def parse_mets_with_metsrw(mets_file):
 
 
 def get_aip_original_name(mets):
-    """Retrieve PREMIS original name from a METSDocument object."""
+    """Retrieve PREMIS original name from a METSDocument object.
+
+    If the original name cannot be reliably retrieved from the METS file
+    a METSError is exception is returned to be handled by the called as
```

Comment:
I think there's a typo in "METSError is exception is returned"

Reply:
Ah, good spot. Thanks Tessa.